packages/wallets/src/components/SentEmailContent/SentEmailContent.tsx

```diff
@@ -72,9 +72,10 @@ const SentEmailContent: React.FC<TProps> = ({ description, platform }) => {
                             setShouldShowResendEmailReasons(true);
                         }}
                         size={emailLinkSize}
-                        text="Didn't receive the email?"
                         variant='ghost'
-                    />
+                    >
+                        Didn&apos;t receive the email?
```

Comment:
Why?
```suggestion
                        Didn't receive the email?
```

Reply:
It causes and ESLint error where it won't let you add apostrophes or any symbols unless it is wrapped with a `' '`